.github/workflows/zeebe-ci.yml

```diff
@@ -282,6 +282,33 @@ jobs:
           secret_vault_secretId: ${{ secrets.VAULT_SECRET_ID }}
           secret_vault_address: ${{ secrets.VAULT_ADDR }}
           secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
+  protobuf-checks:
+    name: "Protobuf Backwards Compatibility"
+    timeout-minutes: 25
+    runs-on: ubuntu-latest
+    if: |
+      contains(fromJSON('["pull_request","push"]'), github.event_name) &&
+      !contains(github.event.pull_request.labels.*.name, 'breaking/protobuf')
```

Comment:
:question: This last expression for the PR label will just treat `push` events as not labeled with `breaking/protobuf`, right? So the check runs for push events.

Reply:
Yes, it will evaluate to false in the case of push, then inverted to true :) So for push it should just work. Of course, this is assumption as I couldn't test yet with push.